### DISCUSSION TITLE
Can the "previous values" drop-down be disabled in data browser?

### DISCUSSION THREAD

**Discussion by argiopetech** [sqlitebrowser/sqlitebrowser#3743] [Q&A] [answered]
When inserting new records in the "Browse Data" pane, columns without constraints show a drop-down menu on tabbing in. This requires an additional key press to dismiss. 

For example, `tab`12`tab` should optimally enter 12 into a column and continue to the next, but, if 12 has previously been entered in that column, will instead highlight 12 in the drop-down menu (requiring `enter tab` at the end of each column entry).

Platform is OS X in case it's platform-dependent.

Is this able to be disabled/worked around?

**Comment by justinclift** [comment, marked as answer]
Uh oh.  That sounds like a bug we'll want to fix in the 3.13.1 release.

@lucydodo Any interest in looking at this? :smile:

**Reply by lucydodo** [reply]
I was able to reproduce the issue. The UX is awkward, even for me. Let me take a look at it.

**Comment by lucydodo** [comment]
This bug was fixed by the 3aff8c9 commit. Can you please confirm with a [continuous build](https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous)?

**Reply by argiopetech** [reply]
Confirmed, drop-down drops down to show previous values, but tab moves to the next column.
